model.py

```diff
@@ -9511,11 +9511,28 @@ def entries_for_work(self, work_or_edition):
             edition = work_or_edition
             work = edition.work
 
-        equivalents = edition.equivalent_editions().all()
+        equivalent_ids = [x.id for x in edition.equivalent_editions()]
```

Comment:
It's weird that these changes showed up again.

Reply:
Yeah, this was in a branch based on my previous branch, but I assumed the commits would disappear from this PR once I merged that branch.